### ISSUE TITLE
No module named 'request' (from panda import PandaRequest)

### ISSUE THREAD

**Issue by nolimitech** [pandas-dev/pandas#15298]
#### Code Sample, a copy-pastable example if possible
```python
# Your code here
import panda as pd
Traceback:
File "/usr/local/lib/python3.4/dist-package/panda/__init__.py", line 1, in <module>
    from request import PandaRequest
```
#### Problem description
I'm working with python 3.4.2, gcc 4.9.1

I have installed everything with pip, all the requirement is good. All the version is good and up to date (my pip3.4 and setuptools, is up to date too).
I've installed Panda and Pandas. (the same)
I've tried to install request, requests (already had)

Don't really know, what to do now :)

#### Expected Output
I can't enter inside the module itself. I can't event know the version.
#### Output of ``pd.show_versions()``

Thanks to help me 

NLT

**Comment by jreback** [comment]
well you are importing ``panda``, not sure what that is. I would say you have an actual directory named ``panda`` with only an ``__init__`` inside.

**Comment by sinhrks** [comment]
maybe refferring to a different package.

https://pypi.python.org/pypi/panda/

**Comment by nolimitech** [comment]
I've tried the apt-get install python3-pandas and it's already installed.
My script name is hello.py, doesn't work in console and IDLE either.
I see the request.py inside the panda directory.
I've found the problem. I had to put a dot before request , model and upload_session modules importation.
``
from .request import..
from .model import..
from .upload_session import ..
``
And i've to add a dot inside the upload_session.py for
``
from .models import Video
``
Now i have a warning error 💃  but inside my home/.local
``
Warning (from warning module):
   File "/home/nlt/.local/lib/python3.4/site-packages/cffi/model.py, line 532
       % self._get_c_name())
Userwarning: 'point_conversion_form_t' has no values explicitly definied; guessing that it is equivalent to 'unsigned int'
``
But everything is working now, only this warning.
(I think, it's working, but i can't find pd.show_versions()

I'll try another package, i think something is broken.

Thanks
